### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by jwilk**
Found using [mwic](http://jwilk.net/software/mwic) and [anorack](https://github.com/jwilk/anorack).

### CODE REVIEW

File: doc/generate_logos.py [hunk 1]
```diff
@@ -243,9 +243,9 @@ def get_svg_filename_from_versionkey(fn_source, ver):
 
 def searchElementById(node, Id, tagname):
     """
-    Search element by id in the all childs and descendant of node.
+    Search element by id in the all children and descendant of node.
```
Comment (smichr) [inline doc/generate_logos.py]: the all -> all the

File: doc/generate_logos.py [hunk 2]
```diff
@@ -243,9 +243,9 @@ def get_svg_filename_from_versionkey(fn_source, ver):
...
 
-    id is lower case, not ID which is usually used for gelElementById
+    id is lower case, not ID which is usually used for getElementById
```
Comment (smichr) [inline doc/generate_logos.py]: getElementById -> getElementsById

File: doc/src/modules/index.rst [hunk 3]
```diff
@@ -11,7 +11,7 @@ Most of the things are already documented though in this document, that is
 automatically generated using SymPy's docstrings.
 
 Click the  "modules" (:ref:`modindex`) link in the top right corner to easily
-access any SymPy module, or use this contens:
+access any SymPy module, or use this contents:
```
Comment (smichr) [inline doc/src/modules/index.rst]: this -> these ?

**Comment by smichr** [comment]
I haven't used those tools before. They look to be effective. How did you like working with them?

File: doc/generate_logos.py (hunk 2 above)
Comment (jwilk) [inline doc/generate_logos.py]: id is supposed to be unique, so the function returns just a single element:
https://developer.mozilla.org/en-US/docs/Web/API/Document/getElementById

File: doc/src/modules/index.rst (hunk 3 above)
Comment (jwilk) [inline doc/src/modules/index.rst]: Yeah, dictionaries say that "contents" is plural, but somehow "these contents" sounds awkward to me. (I'm not a native speaker of English.)
How about making it "this table of contents"?

File: doc/generate_logos.py (hunk 1 above)
Comment (siefkenj) [inline doc/generate_logos.py]: descendant -> descendants

File: doc/src/modules/index.rst (hunk 3 above)
Comment (siefkenj) [inline doc/src/modules/index.rst]: you could say "use this list" instead

File: doc/generate_logos.py (hunk 2 above)
Comment (smichr) [inline doc/generate_logos.py]: ok

File: doc/src/modules/index.rst (hunk 3 above)
Comment (smichr) [inline doc/src/modules/index.rst]: "or use the list below:"

File: doc/src/modules/physics/unitsystems/philosophy.rst [hunk 4]
```diff
@@ -253,7 +253,7 @@ It is not possible to define from scratch units and unit systems: one needs
 to define some references, and then build the rest over them. Said in another
 way, we need an origin for the scales of our units (i.e. a unit with factor 1),
 and to be sure that all units of a given dimension are defined consistently we
-need to use the same origin for all of them. This can happend if we want to use
+need to use the same origin for all of them. This can happened if we want to use
```
Comment (siefkenj) [inline doc/src/modules/physics/unitsystems/philosophy.rst]: happened -> happen

**Review by siefkenj — APPROVED** [review]

**Pull request merged**